Cargo.toml

```diff
@@ -37,9 +37,11 @@ tokio = { version = "1.1.1", features = ["full"] , optional = true}
 tokio-util = { version = "0.6.3", features = ["compat"], optional = true }
 async-std = { version = "1.4.0", features = ["attributes"], optional = true }
 neovim-lib = { version = "0.6.1", optional = true }
+parity-tokio-ipc = { version = "0.9.0", optional = true }
 
 [dev-dependencies]
 tempdir = "0.3"
+rand = "0.4.6"
```

Comment:
`rand` of that version is in the dep graph because of `tempdir` already, I checked while adding it. It was going to be compiled anyway ;-)

I wanted to avoid a potential concurrency/other issue because it's in a global namespace; on Unix it is a hardcoded path relative to a randomly named directory, rather than in a global flat namespace.

Reply:
Ha, very good! I'm kinda planning to move to tempfile, could you leave a TODO comment here to check the version mismatch when doing that? Thanks! Need to take care we only have 1x `rand` in the dependency tree.

Reply:
Done.